app/controllers/concerns/fastly_headers.rb

```diff
@@ -8,13 +8,20 @@ module FastlyHeaders
   # Surrogate-Control is stripped at the cache, Cache-Control persists (in case of other caches in front of fastly)
   # Defaults are:
   #  Cache-Control: 'public, no-cache'
-  #  Surrogate-Control: 'max-age: 86400' - 30 days in seconds
+  #  Surrogate-Control: 'max-age: 86400' - 1 day in seconds
   # custom config example:
   #  {cache_control: 'public, no-cache, maxage=xyz', surrogate_control: 'max-age: 100'}
-  def set_cache_control_headers(max_age = 1.day.to_i, opts = {})
+  def set_cache_control_headers(
+    max_age = 1.day.to_i, cache_control: "public, no-cache",
```

Comment:
**thought (non-blocking):** This signature with 1 normal arg 4 kwargs, all with defaults and 3 of which get passed on to `build_surrogate_control` makes me believe that a controller concern maybe wasn't the right abstraction for this. Once this PR is done we may want to revisit this topic.

Reply:
@citizen428 @rhymes just dropping in with some quick context on this area specifically...

This was introduced to move off the since deprecated [`fastly-rails` gem](https://github.com/fastly/fastly-rails). We basically extracted the chunks we were using and brought it "in-house". We didn't go too far in trying to refactor and modify the code since it's sensitive and we're in the process of moving off of Fastly.

> a controller concern maybe wasn't the right abstraction for this

I see what you mean. That's how the gem had done it so we did a "1-for-1" copy to ensure it was still working. When we go to move off of Fastly, I'll keep this code in mind and we may improve it.